### PR TITLE
Document all ray commands.

### PR DESCRIPTION
**Pull request by ericl**
## Why are these changes needed?

Add short text help for all the click commands.

### CODE REVIEW

**Comment by ericl** [comment]
Also, does anyone know how to "hide" commands? There are a number of old aliases or deprecated commands that make sense to hide from the help.

**Comment by pcmoritz** [comment]
Yes, you can use the `hidden=True` keyword in click (might only be supported in >= 7.0)

**Comment by richardliaw** [comment]
https://click.palletsprojects.com/en/7.x/api/#click.Command

**Comment by richardliaw** [comment]
This looks good; it'd be nice to surface this more on the docs, though I'm not sure what to do

**Review by richardliaw — APPROVED** [review]

**Comment by anabranch** [comment]
I like this but would prefer hidden commands for the duplicates.

File: python/ray/scripts/scripts.py [hunk 1]
```diff
@@ -942,6 +948,7 @@ def timeline(address):
     type=str,
     help="Override the address to connect to.")
 def stat(address):
+    """Get the current metrics protobuf from a Ray cluster."""
```
Comment (anabranch) [inline python/ray/scripts/scripts.py:951]: I don't understand this one. I understand the words, but is this approach documented anywhere?

**Review by bllchmbrs — APPROVED** [review]
LGTM with a nit

File: python/ray/scripts/scripts.py (hunk 1 above)
Comment (ericl) [reply python/ray/scripts/scripts.py:951]: It's an internal debugging command. I updated the comment to reflect this.

**Comment by ericl** [comment]
Yeah hiding doesn't seem to work (and I'm on latest click).

**Comment by pcmoritz** [comment]
Hm, it works for me like so:

```
@click.group("something", help="...", hidden=True)

@something_cli.command(
    name="hidden_command", help="...", hidden=True
)
```

**Review by robertnishihara — APPROVED** [review]
Looks good to me.

**Comment by ericl** [comment]
Got it, the only issue is we want to hide aliases of a command, which there isn't a way to do. It's all or nothing.

**Pull request merged**